### PR TITLE
WEBITOOLS-227: Changes filename from uuid to name

### PR DESCRIPTION
**Pull request by CAST-ANJ**

### CODE REVIEW

File: aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/UploadServiceImpl.java [hunk 1]
```diff
@@ -83,8 +82,9 @@ public String uploadFileForOnboarding(File filePath, String applicationGuid) thr
         String sourcePath;
         final String[] uploadedFilePath = new String[1];
         String archiveExtension = com.castsoftware.aip.console.tools.core.utils.FilenameUtils.getFileExtension(filePath.getName());
+        String fileName = com.castsoftware.aip.console.tools.core.utils.FilenameUtils.getFileName(filePath.getName());
```
Comment (leon2Meudon92) [inline aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/UploadServiceImpl.java:85]: Please also check when filename passed as an environment variable

File: aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/utils/FilenameUtils.java [hunk 2]
```diff
@@ -16,4 +13,14 @@ public static String getFileExtension(String filename) {
         }
         return org.apache.commons.io.FilenameUtils.getExtension(filename);
     }
+
+    public static String getFileName(String filename) {
```
Comment (leon2Meudon92) [inline aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/utils/FilenameUtils.java:17]: try org.apache.commons.io.FilenameUtils instead

File: aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/UploadServiceImpl.java (hunk 1 above)
Comment (CAST-ANJ) [reply aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/UploadServiceImpl.java:85]: filename is not present in env variable. only file path can be provided.

File: aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/utils/FilenameUtils.java (hunk 2 above)
Comment (CAST-ANJ) [reply aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/utils/FilenameUtils.java:17]: Changes done.

**Pull request merged**